src/DynamoRevit/DynamoRevitApp.cs

```diff
@@ -250,5 +250,11 @@ public static Assembly ResolveAssembly(object sender, ResolveEventArgs args)
                 throw new Exception(string.Format("The location of the assembly, {0} could not be resolved for loading.", assemblyPath), ex);
             }
         }
+
+        public static bool DynamoButtonEnabled
```

Comment:
If this property is not supposed to be used by other modules, s\public\internal.